src/CMakeLists.txt

```diff
@@ -290,6 +290,7 @@ if(NOT fmt_FOUND)
   message(STATUS "Could not find fmt, will build it")
   add_subdirectory(fmt)
 endif()
+include_directories(SYSTEM "${CMAKE_SOURCE_DIR}/src/fmt/include")
```

Comment:
why? to silence the warning on aarch64?

Reply:
To ensure we could access fmt from all projects without having to include it specifically fo reach one.

Reply:
(I've been replacing uses of string streams or concatenating lots of strings together with it, when I see them.)